.gitignore

```
@@ -0,0 +1,4 @@
+/target
+/node_modules
+
+Cargo.lock
```

```suggestion
Cargo.lock
```
lockファイルはlibではignoreするべきとされているのですが、今回のケースは特殊なのでbinと同じ扱い(lockファイルもコミットする)でいいと思います。

---

あっやろうとおもってわすれてました

---

12f68ca